app/resources/documentation/manual/index.md

```diff
@@ -935,7 +985,15 @@ To rotate the texture about the origin, left click and drag the large yellow cir
 
 Shearing the texture is possible if the map uses a [parallel texture projection](#texture_projection_modes). Shear by holding #key(Alt) while left clicking and dragging the gray texture grid lines.
 
-At the bottom of the UV editor, you can find a number of controls. On the left, there are five buttons. The first button resets the face attributes to their defaults. The second and third buttons flip the texture horizontally and vertically, respectively, and the fourth and firth buttons rotate the texture by 90° counter clockwise and clockwise. The two controls on the right allow you to subdivide the texture grid. This can be useful to align the texture to smaller brush faces.
+At the bottom of the UV editor are the following controls:
+
```

Comment:
I feel like a screenshot of the UI could be really helpful here.

Reply:
Added:
<img width="618" alt="Screen Shot 2021-01-27 at 7 13 10 PM" src="https://user-images.githubusercontent.com/239161/106080142-bff80300-60d3-11eb-9ef0-de0fac1f82f8.png">